docs/google-cloud.md

```diff
@@ -34,107 +44,121 @@ gcloud auth configure-docker
-1. If the VMs consume too little CPU, Kubernetess will shut down some VMs.
-1. Some very expensive challenges will probably be allocated in the same VMs as less busy challenges.
-
-This all ensures the availability of the challenges, and saves some computing resources.
+1. If the VMs consume too little CPU, Kubernetes will shut down some VMs.
+1. Some very expensive challenges can be allocated to the same VMs as less busy challenges.
 
-Anyway, hopefully by now the cluster is already created and we can continue with the walkthrough.
+The above steps ensure the availability of the challenges, while using computing resources in a resourceful manner.
 
-If you are curious and have some more time, take a look at this [kCTF introduction](introduction.md), which includes a quick 8 minutes summary of what is kCTF and how it interacts with Kubernetes.
+By now, your cluster should be ready and we can continue with the walkthrough.
 
-# Step 2 - Create a challenge
-Now that we have a cluster setup, we need to create a challenge.
+Note: If you are curious and have some spare time, take a look at the [kCTF introduction](https://github.com/google/kctf/blob/master/docs/introduction.md), which includes a quick 8 minute summary of what kCTF is and how it interacts with Kubernetes.
 
-In this walkthrough, you'll learn how to create a challenge called "demo-challenge", build the docker image, deploy it to the cluster, and expose it to the internet.
+# Step 2 – Create a challenge
+Now that you have set up a cluster, you can create a challenge.
 
-You need the cluster to be created before you continue, otherwise the following commands won't work.
+In this step, you'll learn how to:
+* Create a challenge called "demo-challenge"
+* Build a Docker image
+* Deploy the Docker image to the cluster
+* Expose the Docker image to the internet
 
-Click next to continue if your cluster is already created.
+Note: The cluster must be created before you continue, otherwise the following commands won't work. To create a cluster, see the preceding steps. Continue with the next steps if you already created a cluster.
 
-## First, call create_challenge.sh to copy the skeleton
-Run the following command to create a challenge called "demo-challenge"
+## Call create_challenge.sh to copy the skeleton
+Run the following command to create a challenge called "demo-challenge":
 ```
 kctf-chal-create demo-challenge
 ```
 
-This will create a directory called `demo-challenge` under the `kctf-demo` directory, and if you look inside of it (check files/chal for example), you'll find out the challenge configuration. The file in `files/chal` is the entry-point, that means it is executed every time there's a TCP connection. This demo challenge just runs bash, but a real challenge would instead expose a harder challenge that doesn't just give out a shell.
+This creates a directory called `demo-challenge` under the `kctf-demo` directory.
```

Comment:
the directory is not kctf-demo atm, its demo-ctf-cluster. we need to change them so they work

Reply:
sorry, the implication is not clear to me. Should I change the doc to "demo-ctf-cluster", or will you change the solution to use "kctf-demo"?

Reply:
I mean, the documentation was wrong before, and I just noticed =). We should make sure we use the same name for the kCTF walkthrough demo everywhere.

In other words, the step here (`~/kctf-demo`):
https://github.com/google/kctf/blob/5db06f7a02d93359753c4e50d964d78cd324b352/docs/google-cloud.md#set-configuration-properties

Should match the step here (`~/demo-ctf-cluster`):
https://github.com/google/kctf/blob/5db06f7a02d93359753c4e50d964d78cd324b352/docs/google-cloud.md#deploy-the-challenge